packages/core/src/components/autosuggest/autosuggest.scss

```diff
@@ -35,9 +36,12 @@ ul {
   padding: units.$u1 0;
 
   li {
-    display: flex;
-    justify-content: space-between;
     padding: units.$u1 * 0.5 units.$u2;
+
+    div {
```

Comment:
Ik zou lekker 'n class gebruiken, maakt het in de JSX ook duidelijk dat het element buiten de `.map()` van hetzelfde soort als in de `.map()` is.

`.suggestion-row`, oid.

Reply:
✅